### DISCUSSION TITLE
Welcome to binance-trading-bot-new-coins Discussions!

### DISCUSSION THREAD

**Discussion by CyberPunkMetalHead** [CyberPunkMetalHead/binance-trading-bot-new-coins#41] [Announcements]
## 👋 Welcome! Discussions are now enabled.
  * Stay up to date with the progress.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CyberPunkMetalHead** [comment]
Hey guys. Just a note to let you all know that I've tested on my local environment that placing sell orders is a lot more profitable than buy orders. I'll soon push a change that will execute margin sell orders instead but I'm still trying to figure out how this will all work since not all new coins are listed on margin/futures at the beginning.

If you have any ideas on how this could be achieved, please let me know!

**Comment by CyberCyclone** [comment]
I noticed that the default configs are a lot more conservative to the ones you used in the demo. Is the default margin / percentages in the repo your recommended values? Based on your experience, do you adjust them? It would be good to know the values you use. Thanks!

P.S, a good way to run the scripts in the background is to use a nodejs plugin call 'pm2'. It also handles crashes and restarts the script.
